x/evm/client/cli/tx.go

```diff
@@ -103,3 +107,88 @@ func CmdAssociateAddress() *cobra.Command {
 
 	return cmd
 }
+
+func CmdSend() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "send [to EVM address] [amount in usei] [nonce]",
+		Short: "send usei to EVM address",
+		Long:  "",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			txf := tx.NewFactoryCLI(clientCtx, cmd.Flags())
+			kb := txf.Keybase()
+			info, err := kb.Key(clientCtx.GetFromName())
+			if err != nil {
+				return err
+			}
+			localInfo, ok := info.(keyring.LocalInfo)
+			if !ok {
+				return errors.New("can only associate address for local keys")
+			}
+			priv, err := legacy.PrivKeyFromBytes([]byte(localInfo.PrivKeyArmor))
+			if err != nil {
+				return err
+			}
+			privHex := hex.EncodeToString(priv.Bytes())
+			key, _ := crypto.HexToECDSA(privHex)
+
+			to := common.HexToAddress(args[0])
+			val, err := strconv.ParseUint(args[1], 10, 64)
+			if err != nil {
+				return err
+			}
+			nonce, err := strconv.ParseUint(args[2], 10, 64)
+			if err != nil {
+				return err
+			}
+			txData := ethtypes.DynamicFeeTx{
+				Nonce:     nonce,         // TODO query from server
+				GasFeeCap: big.NewInt(1), // TODO use flag
+				Gas:       21000,         // TODO use flag
+				To:        &to,
+				Value:     new(big.Int).Mul(new(big.Int).SetUint64(val), state.UseiToSweiMultiplier),
+				Data:      []byte(""),
+				ChainID:   big.NewInt(713715), // TODO query from server
+			}
+			ethCfg := types.DefaultChainConfig().EthereumConfig(txData.ChainID)
+			signer := ethtypes.MakeSigner(ethCfg, big.NewInt(1), 1)
+			tx := ethtypes.NewTx(&txData)
+			tx, err = ethtypes.SignTx(tx, signer, key)
+			if err != nil {
+				return err
+			}
+			bz, err := tx.MarshalBinary()
+			if err != nil {
+				return err
+			}
+			payload := "0x" + hex.EncodeToString(bz)
+
+			body := fmt.Sprintf("{\"jsonrpc\": \"2.0\",\"method\": \"eth_sendRawTransaction\",\"params\":[\"%s\"],\"id\":\"send\"}", payload)
+			req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("http://%s:8545", evmrpc.LocalAddress), strings.NewReader(body))
```

Comment:
should we allow the user to specify their own rpc endpoint?

Reply:
updated